test/extended/util/framework.go

```diff
@@ -297,6 +297,11 @@ var CheckPodIsRunningFunc = func(pod kapi.Pod) bool {
 	return pod.Status.Phase == kapi.PodRunning
 }
 
+// CheckPodIsSucceededFunc returns true if the pod status is "Succdeded"
+var CheckPodIsSucceededFunc = func(pod kapi.Pod) bool {
```

Comment:
Clayton, Paul, 

yup, no problem. I'm going to modify  `*Func` to `*Fn`. Thanks for the info